src/components/molecules/Common/AssetModal/VirtualAssetCard/index.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useCallback, useState, useEffect } from "react";
+import { styled } from "@reearth/theme";
+
+type CardSize = "small" | "medium" | "large" | "list";
+
+export type Props = {
+  assetsLength?: number;
+  containerWidth?: number;
+  cardSize?: CardSize;
+};
+
+const VirtualAssetCard: React.FC<Props> = ({ assetsLength, containerWidth, cardSize }) => {
+  const [virtualWidth, setVirtualWidth] = useState(0);
+
+  const handleVirtualWidth = useCallback(() => {
+    if (containerWidth) {
+      if (assetsLength) {
+        const cardWidth = cardSize === "medium" ? 216 : 128;
```

Comment:
I'm a little concerned that the magic numbers (216 and 128) is hard-coded this way. It is easier to understand if you define it as a const outside the component.

Reply:
sure, thanks